src/WebUI/Common/ContentReader.tsx

```
@@ -81,7 +81,11 @@ export class ContentReader extends React.Component<IReaderProps> {
 
     private _disposeEditor(): void {
         if (this._editor) {
-            this._editor.dispose();
+            try {
+                this._editor.dispose();

```

when disposing the editor we got an error, so swallowing the same